pkg/webui/console/containers/notifications/index.js

```diff
@@ -0,0 +1,149 @@
+// Copyright © 2023 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React, { useCallback, useEffect, useState } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { Col, Row } from 'react-grid-system'
+import classNames from 'classnames'
+import { defineMessages } from 'react-intl'
+
+import Pagination from '@ttn-lw/components/pagination'
+import Button from '@ttn-lw/components/button'
+
+import attachPromise from '@ttn-lw/lib/store/actions/attach-promise'
+import PropTypes from '@ttn-lw/lib/prop-types'
+import useQueryState from '@ttn-lw/lib/hooks/use-query-state'
+
+import { getNotifications } from '@console/store/actions/notifications'
+
+import { selectUserId } from '@console/store/selectors/logout'
+import { selectTotalNotificationsCount } from '@console/store/selectors/notifications'
+
+import NotificationList from './notification-list'
+import NotificationContent from './notification-content'
+
+import style from './notifications.styl'
+
+const m = defineMessages({
+  seeArchived: 'See archived messages',
+  seeAll: 'See all messages',
+})
+
+const pageSize = 5
+const DEFAULT_PAGE = 1
+
+const pageValidator = page => (!Boolean(page) || page < 0 ? DEFAULT_PAGE : page)
+
+const NotificationsContainer = ({ setPage, page }) => {
```

Comment:
```suggestion
const Notifications = ({ setPage, page }) => {
```

Not really conventional to add `Container` to the component name.